### PR TITLE
Update metrics docs for JS

### PR DESCRIPTION
**Pull request by svrnm**
Contributes to https://github.com/open-telemetry/opentelemetry.io/issues/3229, this updates the first few sections for the instrumentation guidelines for JavaScript.

---

**Preview**:

- https://deploy-preview-3853--opentelemetry.netlify.app/docs/concepts/signals/metrics
- https://deploy-preview-3853--opentelemetry.netlify.app/docs/languages/js/instrumentation

### CODE REVIEW

File: content/en/docs/concepts/signals/metrics.md
Comment (theletterf) [inline content/en/docs/concepts/signals/metrics.md:77]: I miss an introductory sentence here that presents the two possible types of instruments. The order should also be the same as the title, and so start with synchronous and then follow with async.

File: content/en/docs/concepts/signals/metrics.md
Comment (theletterf) [inline content/en/docs/concepts/signals/metrics.md:80]:
```suggestion
- When updating a counter is not computationally cheap and you don't want the
```

File: content/en/docs/concepts/signals/metrics.md
Comment (theletterf) [inline content/en/docs/concepts/signals/metrics.md:83]:
```suggestion
  (for example, they can't be accurately measured when tied to a 
  request lifecycle)
```

File: content/en/docs/concepts/signals/metrics.md
Comment (theletterf) [inline content/en/docs/concepts/signals/metrics.md:89]:
```suggestion
In the previous cases, it's often better to observe a cumulative value directly
rather than aggregate a series of deltas in post-processing (the synchronous
example). Note here the use of `observe` instead of `add` in the following
examples.
```

Comment (theletterf) [reply content/en/docs/concepts/signals/metrics.md:89]: Which examples?

File: content/en/docs/concepts/signals/metrics.md
Comment (theletterf) [inline content/en/docs/concepts/signals/metrics.md:95]:
```suggestion
an export cycle can contain zero or multiple aggregated measurements.
```

File: content/en/docs/languages/js/instrumentation.md
Comment (theletterf) [inline content/en/docs/languages/js/instrumentation.md:1230]:
```suggestion
To learn more about the differences between synchronous and
asynchronous instruments, and when to use each kind, see the
concept page on
```

**Review by theletterf — COMMENTED** [review]
Thanks! Just some style comments and nits.

File: content/en/docs/concepts/signals/metrics.md
Comment (cartermp) [reply content/en/docs/concepts/signals/metrics.md:90]: I would wager few outside of those who contribute to OTel would understand this sentence:

> In cases like these, it's often better to observe a cumulative value directly,
rather than aggregate a series of deltas in post-processing (the synchronous
example).

Is it really needed?

**Review by cartermp — COMMENTED** [review]
Overall looks fine. @open-telemetry/javascript-approvers PTAL

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1197]: "aggregations" --> "aggregates"?

```suggestion
[Metrics](/docs/concepts/signals/metrics) combine individual measurements into
aggregates, and produce data which is constant as a function of system load.
Aggregates lack details required to diagnose low level issues, but
```

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1206]:
```suggestion
Synchronous instruments record measurements as they happen. Asynchronous
instruments register a callback that is invoked once per collection, and which
record measurements at that point in time.
```

"at that point in time" isn't as clear as it could be, but it's good enough for now.

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1226]:
```suggestion
{{% alert title="Note" class="info" %}} OpenTelemetry instruments are either
```

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1233]:
```suggestion
{{% /alert %}}
```

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1224]: Could we consolidate this sentence with the alert you added just below. It doesn't make sense to have a normal paragraph refer the reader to the spec via: "For more on synchronous and asynchronous instruments, and which kind is best suited for your use case, see [Supplementary Guidelines](/docs/specs/otel/metrics/supplementary-guidelines/)" immediately followed by an alert that says ~ "for more info see the concept page".

File: content/en/docs/languages/js/instrumentation.md
Comment (chalin) [inline content/en/docs/languages/js/instrumentation.md:1294]: A general comment since it has been coming up lately: to me it's a bad smell that we're repeating the default value in several places in the docs. This will be a pain to update if the default happens to change. How about something like:

```suggestion
  exportIntervalMillis: 5000, // Reduce export interval for demo purposes only.
```

I'd rather that we link to a page that says what the default is instead of repeating the default value.

Comment (chalin) [reply content/en/docs/languages/js/instrumentation.md:1294]: Also, I think that we agreed elsewhere that this would be ~ 10 sec?

File: content/en/docs/concepts/signals/metrics.md
Comment (svrnm) [inline content/en/docs/concepts/signals/metrics.md:1]: @theletterf @chalin @cartermp just to add some context: the changes to this file are coming from some text I moved out from the JS docs to the general docs, I may not have all the knowledge about all the text if & how it makes sense, so if you have any suggestions to make that text better, provide me with inline suggestions:-)

File: content/en/docs/concepts/signals/metrics.md
Comment (svrnm) [reply content/en/docs/concepts/signals/metrics.md:90]: > Which examples?

the JS examples, so that sentence makes no sense anymore

Comment (svrnm) [reply content/en/docs/concepts/signals/metrics.md:90]:
```suggestion
```

Should I remove that entirely, wdyt?

File: content/en/docs/languages/js/instrumentation.md
Comment (svrnm) [reply content/en/docs/languages/js/instrumentation.md:1224]: ah, oh, I didn't see that. This may also fix some of the things above.

File: content/en/docs/languages/js/instrumentation.md
Comment (svrnm) [inline content/en/docs/languages/js/instrumentation.md:1233]:
```suggestion
```

File: content/en/docs/languages/js/instrumentation.md
Comment (svrnm) [reply content/en/docs/languages/js/instrumentation.md:1294]: > in the PR for java who discusses to introduce that

Also, I think that we agreed elsewhere that this would be ~ 10 sec?

Comment (svrnm) [reply content/en/docs/languages/js/instrumentation.md]: let's have a dedicated issue to make sure that we have some consistency here.

File: content/en/docs/concepts/signals/metrics.md
Comment (svrnm) [reply content/en/docs/concepts/signals/metrics.md]: I removed a lot of that content and point to the guidelines from the spec which have all that we need.

**Comment by svrnm** [comment]
@open-telemetry/docs-approvers , @open-telemetry/javascript-approvers please take another look. I removed a lot of the conceptual content from the "instrumentation" page since this belongs to the concepts page (and there I point to the guideline in the specs)

**Review by chalin — APPROVED** [review]
Overall LGTM. I'll let the others do a more thorough review.

**Review by cartermp — APPROVED** [review]
Similarly a ✅ from me, including after the changes since I last reviewed

**Comment by svrnm** [comment]
@open-telemetry/javascript-approvers PTAL!

**Review by pichlermarc — APPROVED** [review]
This looks good, thanks for working on this @svrnm

**Comment by svrnm** [comment]
> This looks good, thanks for working on this @svrnm

thanks for taking a look!

**Pull request merged**
